main.py

```diff
@@ -1113,11 +1113,13 @@ def __call__(
         environ: Dict[str, str],
         start_response: webapp2.Response
     ) -> webapp2.Response:
+        # Here we use MyPy ignore because cache_services is of ModuleType
+        # and ModuleType does not contain 'CLOUD_NDB_REDIS_CLIENT' attribute.
```

Comment:
I don't quite follow this. Don't we typically import top-of-file constants in other places? If so, why is this the only one that needs a MyPy ignore?

Reply:
Fixed